client/src/shared/components/InputField/index.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable object-shorthand */
+import React from 'react';
+
+interface InputFieldProps {
```

Comment:
- this interface could be usable in the future development. so please export it.
```suggestion
export interface InputFieldProps {
```

**can you kindly apply the same thing as well in the other existing components**